src/processor/graph-update-completion-monitor.service.ts

```diff
@@ -13,12 +13,12 @@ import * as ReconnectionServiceConstants from '#app/constants';
 @Injectable()
 export class GraphUpdateCompletionMonitorService extends BlockchainScannerService implements OnApplicationBootstrap, OnApplicationShutdown {
   async onApplicationBootstrap() {
-    const pendingTxns = await this.cacheManager.getAllPendingTxns();
+    //const pendingTxns = await this.cacheManager.getAllPendingTxns();
     // If no transactions pending, skip to end of chain at startup
-    if (Object.keys(pendingTxns).length === 0) {
-      const blockNumber = await this.blockchainService.getLatestFinalizedBlockNumber();
-      await this.setLastSeenBlockNumber(blockNumber);
-    }
+    // if (Object.keys(pendingTxns).length === 0) {
+    //   const blockNumber = await this.blockchainService.getLatestFinalizedBlockNumber();
+    //   await this.setLastSeenBlockNumber(blockNumber);
+    // }
```

Comment:
This logic is not useful but can cause a chain to move scanning to latest block given there might not be pending txs at start of the service while we have many blocks to scan